packages/big-design/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { HTMLAttributes, memo } from 'react';
+
+import { Flex } from '../Flex';
+
+import { Step } from './Step';
+
+export interface StepperProps extends HTMLAttributes<HTMLDivElement> {
+  steps: Array<string>;
+  currentStep: number;
+}
+
+export const Stepper: React.FC<StepperProps> = memo(({ className, style, steps, currentStep, ...props }) => (
```

Comment:
Instead of providing defaultProps below, we may want to inline them here. Maybe even exclude `steps` having a default prop. Also, we may want to check if the provided `steps` array has length to ensure we don't render anything if an empty array is provided. The main reasoning for all of this is to guard rendering in non-Typescript projects + empty arrays.
```suggestion
export const Stepper: React.FC<StepperProps> = memo(({ className, style, steps, currentStep = 0, ...props }) => steps.length ? (
// make sure to finish the ternary
```